test/test_apps/external-memory-fd/import/app.cpp

```diff
@@ -274,8 +285,14 @@ bool App::frame(const int frame_num)
 
 void App::cleanup()
 {
+    usleep(30000);
```

Comment:
I'd prefer something like the import app holds the socket open until `cleanup`, then the export app `read`s the socket or `select`s on the socket after sending the fd and waits for the `read` to return 0 bytes, which it should do when the import app closes the socket.  Then no worries about race condition.

(I'm surprised there isn't a reference count on that memory but I can't find anythign in the spec about it.)